app/backend/requirements.in

```diff
@@ -7,7 +7,7 @@ tiktoken
 tenacity
 azure-ai-documentintelligence
 azure-cognitiveservices-speech
-azure-search-documents==11.6.0b1
+azure-search-documents==11.6.0b6
```

Comment:
I didn't get any errors about code changes on the .search() side so I'm assuming that's largely unchanged in the latest versions?

Reply:
correct